packaging/dbscripts/upgrade/pre_upgrade/0000_config.sql

```diff
@@ -824,6 +824,7 @@ select fn_db_add_config_value_for_versions_up_to('BiosTypeSupported', 'true', '4
 select fn_db_add_config_value('LiveSnapshotTimeoutInMinutes', '30', 'general');
 select fn_db_add_config_value('LiveSnapshotAllowInconsistent', 'true', 'general');
 select fn_db_add_config_value('LiveSnapshotFreezeTimeout', '8', 'general');
+select fn_db_add_config_value('VolumeUtilizationChunkInMB', '2560', 'general');
```

Comment:
it makes sense to add it in 'general' scope because it's not (and can't be) per-cluster level in vdsm and we don't use it in cluster levels < 4.7, but it may still confuse someone that would try to use it regardless of the cluster level - how about adding a comment to the config value that explains it reflects the value in VDSM 4.7?

Reply:
Generally I have nothing against comments in code :)
Just a few thoughts, why (in this case) I'm not sure about adding comments:
1) Consistency with the rest of the code - I don't see comments describing other configurations, only section titles. And I'm not sure this new configuration is "so special".
2) Many other new configurations were also cluster-level-specific and they don't have comments describing that, but require to dig into code. So, again, having some doubts...

Reply:
> 1. Consistency with the rest of the code - I don't see comments describing other configurations, only section titles. And I'm not sure whether this new configuration is "so special".

what makes this one special is that it's a config value in the engine that should align with VDSM but VDSM before 4.5 had a different value and this one is in "general" scope - it's quite a unique situation

> 2. Many other new configurations were also cluster-level-specific and they don't have comments describing that, but require to dig into code. So, again, having some doubts...

we can go this way and make it CL-specific but then what would we put in CL < 4.7? we anyway don't use it in that case..

Reply:
@ahadas Please check the new version. Added a comment, tried to be consistent with the existing code style.